includes/organization_config.php

```diff
@@ -11,52 +11,52 @@ function organization_config_mnu() {
+		$org_options['show_pending_payment_options'] = isset($_POST['show_pending_payment_options']) && !empty($_POST['show_pending_payment_options']) ? sanitize_text_field($_POST['show_pending_payment_options']) : '';
+		$org_options['use_venue_manager'] = isset($_POST['use_venue_manager']) && !empty($_POST['use_venue_manager']) ? sanitize_text_field($_POST['use_venue_manager']) : '';
+		$org_options['use_personnel_manager'] = isset($_POST['use_personnel_manager']) && !empty($_POST['use_personnel_manager']) ? sanitize_text_field($_POST['use_personnel_manager']) : '';
+		$org_options['use_event_timezones'] = isset($_POST['use_event_timezones']) && !empty($_POST['use_event_timezones']) ? sanitize_text_field($_POST['use_event_timezones']) : '';
+		$org_options['full_logging'] = isset($_POST['full_logging']) && !empty($_POST['full_logging']) ? sanitize_text_field($_POST['full_logging']) : '';
+		$org_options['surcharge'] = isset($_POST['surcharge']) && !empty($_POST['surcharge']) ? sanitize_text_field($_POST['surcharge']) : '';
+		$org_options['surcharge_type'] = isset($_POST['surcharge_type']) && !empty($_POST['surcharge_type']) ? sanitize_text_field($_POST['surcharge_type']) : '';
+		$org_options['surcharge_text'] = isset($_POST['surcharge_text']) && !empty($_POST['surcharge_text']) ? sanitize_text_field($_POST['surcharge_text']) : '';
+		$org_options['show_reg_footer'] = isset($_POST['show_reg_footer']) && !empty($_POST['show_reg_footer']) ? sanitize_text_field($_POST['show_reg_footer']) : '';
+		$org_options['affiliate_id'] = isset($_POST['affiliate_id']) && !empty($_POST['affiliate_id']) ? sanitize_text_field($_POST['affiliate_id']) : '';
 		$org_options['site_license_key'] = isset($_POST['site_license_key']) && !empty($_POST['site_license_key']) ? trim($_POST['site_license_key']) : '';
-		$org_options['default_payment_status'] = isset($_POST['default_payment_status']) && !empty($_POST['default_payment_status']) ? $_POST['default_payment_status'] : '';
-		$org_options['default_promocode_usage'] = isset($_POST['default_promocode_usage']) && !empty($_POST['default_promocode_usage']) ? $_POST['default_promocode_usage'] : 'N';
-		$org_options['ticket_reservation_time'] = isset($_POST['ticket_reservation_time']) && !empty($_POST['ticket_reservation_time']) ? $_POST['ticket_reservation_time'] : '30';
+		$org_options['default_payment_status'] = isset($_POST['default_payment_status']) && !empty($_POST['default_payment_status']) ? sanitize_text_field($_POST['default_payment_status']) : '';
+		$org_options['default_promocode_usage'] = isset($_POST['default_promocode_usage']) && !empty($_POST['default_promocode_usage']) ? sanitize_text_field($_POST['default_promocode_usage']) : 'N';
+		$org_options['ticket_reservation_time'] = isset($_POST['ticket_reservation_time']) && !empty($_POST['ticket_reservation_time']) ? sanitize_text_field($_POST['ticket_reservation_time']) : '30';
```

Comment:
another int field. So
```suggestion
		$org_options['ticket_reservation_time'] = isset($_POST['ticket_reservation_time']) && !empty($_POST['ticket_reservation_time']) ? (string)intval($_POST['ticket_reservation_time']) : '30';
```